styles.css

```diff
@@ -9,612 +9,965 @@ body {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  /* card 2  */
+
+  .work-container2 {
+    border: 1px solid #cbcdd1;
+    display: flex;
+    flex-direction: row-reverse;
+    width: 75%;
+    height: 580px;
+    background: #fff;
+    border-radius: 16px;
+    justify-content: center;
+    gap: 40px;
+    padding: 20px;
+  }
+
+  .tonic {
+    height: auto;
+    width: auto;
+  }
+
+  .desktop-img-1 {
+    background-image: url("images/Snapshoot-Portfolio-4.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    align-self: center;
+    width: 50%;
+    height: 448px;
+    display: flex;
+    margin-top: 20px;
+  }
+
+  .ruby {
+    border: 2px;
+    display: flex;
+    margin-left: 4px;
+    justify-content: center;
+    align-items: center;
+    padding: 4px 12px;
+    width: 114px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+    font-size: 14px;
+  }
+
+  /* card 3  */
+  .work-container3 {
+    border: 1px solid #cbcdd1;
+    display: flex;
+    width: 75%;
+    height: 580px;
+    background: #fff;
+    border-radius: 16px;
+    justify-content: center;
+    gap: 40px;
+    padding: 20px;
+  }
+
+  .desktop-img-2 {
+    background-image: url("images/Snapshoot-Portfolio.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    align-self: center;
+    width: 50%;
+    height: 448px;
+    display: flex;
+    margin-top: 20px;
+  }
+
+  /* card 4  */
+  .work-container4 {
+    border: 1px solid #cbcdd1;
+    display: flex;
+    flex-direction: row-reverse;
+    width: 75%;
+    height: 580px;
+    background: #fff;
+    border-radius: 16px;
+    justify-content: center;
+    gap: 40px;
+    padding: 20px;
+  }
+
+  .desktop-img-3 {
+    background-image: url("images/Snapshoot-Portfolio-2.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    align-self: center;
+    width: 50%;
+    height: 448px;
+    display: flex;
+    margin-top: 20px;
+  }
+
+  /* about  */
+  .introd {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .end-about {
+    width: 35%;
+    margin: 25px;
+    padding-left: 20px;
+  }
+
+  .end-about h2 {
+    margin-top: 50px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .end-about p {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .about-info br {
+    display: none;
+  }
+
+  .lang {
+    display: flex;
+    flex-direction: column;
+    width: 45%;
+    height: 400px;
+    align-self: center;
+    justify-content: space-evenly;
+    margin-top: -480px;
+    padding-left: 30px;
+  }
+
+  .lang-list {
+    flex-direction: row;
+    gap: 12px;
+    align-items: flex-start;
+  }
+
+  /* contact  */
+  .contact {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
Kindly update the styling of your contact section to better match the figma design🙏.

Your design:
<img width="541" alt="image" src="https://user-images.githubusercontent.com/37116322/220736956-569d5583-931d-411c-b493-085a2ec6c636.png">

Figma design:
<img width="483" alt="image" src="https://user-images.githubusercontent.com/37116322/220737011-e3ada281-0263-4a05-9969-fb30a228d3b7.png">

A few things to note:
- Please increase the width of the main content of the contact section
- Kindly resize your background image so it spans the entire width of the contact section
- Please remove the border radius on the input elements and align the `get in touch` button to the center